### PR TITLE
test: add new test cases for pkg/protocol/multipart.go

### PR DESCRIPTION
**Pull request by gityh2021**
#### What type of PR is this?
test

#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.
测试：为pkg/protocol/multipart.go文件新增测试用例。

#### (Optional) More detail description for this PR(en: English/zh: Chinese).

en: add test cases for the following features for pkg/protocol/multipart.go and the test coverage rate of pkg/protocol/response.go is improved to79.75%.
      1.TestParseMultipartForm: test parse multiform from request, including some error tests and set new request with func SetMultipartFormWithBoundary, two requests are eager to be same.
      2.TestWriteMultipartFormFile: test func TestWriteMultipartFormFile and func AddFile.
zh(optional):

#### Which issue(s) this PR fixes:
https://github.com/cloudwego/hertz/issues/257

### CODE REVIEW

**Comment by gityh2021** [comment]
The test coverage rate of multipart.go is 82.6% in my local test tool.However, it is 68% in codecov. I will optimize it.

**Comment by li-jin-gou** [comment]
> The test coverage rate of multipart.go is 82.6% in my local test tool.However, it is 68% in codecov. I will optimize it.

❤️

**Review by li-jin-gou — DISMISSED** [review]

File: pkg/protocol/multipart_test.go [hunk 1]
```diff
@@ -69,11 +71,167 @@ Content-Type: application/json
 		t.Fatalf("unexpected error: %s", err)
 	}
 
-	if err := WriteMultipartForm(&w, form, "foo"); err != nil {
+	// boundary is too long
+	err = WriteMultipartForm(&w, form, s)
```
Comment (FGYFFFF) [inline pkg/protocol/multipart_test.go:75]: The length of boundary is in the range of [1,70], which can be verified for strings outside this range.

Comment (gityh2021) [reply pkg/protocol/multipart_test.go:75]: done, thx~

**Review by li-jin-gou — APPROVED** [review]

**Comment by li-jin-gou** [comment]
LGTM

**Pull request merged**